BeVolt/BSP/Simulator/Src/BSP_Lights.c

```diff
@@ -16,47 +29,52 @@ void BSP_Lights_Init(void) {
  * @return  None
  */
 void BSP_Light_Toggle(Light signal) {
-    // TODO: Toggle the state for each pin
+	FILE *LightsFile;
+	LightsFile = fopen(Lights, "wb+"); //open file for reading and writing
+	fseek(LightsFile, 0, SEEK_SET); //Go to beginning of file
+	fread(&LEDReg, 2, 1, LightsFile); //read data from file
```

Comment:
I would recommend not storing this as a binary file mostly so we're still able to view the file in .csv format for debugging purposes. Something like this would be good:

```
FILE* fp = fopen(Lights, "w+");
char csv[8];
fgets(csv, 8, fp);
LEDReg = atoi(csv);
```